examples/cors/pom.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2020, 2024 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-se</artifactId>
+        <version>4.1.0-SNAPSHOT</version>
+    </parent>
+    <groupId>io.helidon.examples</groupId>
+    <artifactId>helidon-examples-cors</artifactId>
```

Comment:
Need to add the version here explicitly
`     <version>1.0.0-SNAPSHOT</version>`
Because it is inheriting the version from the parent. Looks like this is the case for these examples:
```
[INFO] Helidon Examples CORS SE 4.1.0-SNAPSHOT ............ SUCCESS [  0.669 s]
[INFO] Helidon Examples GraphQL Basics 4.1.0-SNAPSHOT ..... SUCCESS [  0.242 s]
[INFO] Helidon Examples Integration OCI ATP 4.1.0-SNAPSHOT  SUCCESS [  0.502 s]
[INFO] Helidon Examples Media Support Multipart 4.1.0-SNAPSHOT SUCCESS [  0.349 s]
[INFO] Helidon Examples Quickstart MP 4.1.0-SNAPSHOT ...... SUCCESS [  0.473 s]
[INFO] Helidon Examples Security ABAC 4.1.0-SNAPSHOT ...... SUCCESS [  0.435 s]
[INFO] Helidon Examples Security HTTP Basic Auth with Static Content 4.1.0-SNAPSHOT SUCCESS [  0.415 s]
[INFO] Helidon Examples Security Google Login 4.1.0-SNAPSHOT SUCCESS [  0.415 s]
[INFO] Helidon Examples Security IDCS Login 4.1.0-SNAPSHOT  SUCCESS [  0.251 s]
[INFO] Helidon Examples Security Outbound Override 4.1.0-SNAPSHOT SUCCESS [  0.391 s]
[INFO] Helidon Examples Security No-HTTP programmatic 4.1.0-SNAPSHOT SUCCESS [  0.137 s]
[INFO] Helidon Examples Security SPI Implementation 4.1.0-SNAPSHOT SUCCESS [  0.320 s]
[INFO] Helidon Examples Security Vaults 4.1.0-SNAPSHOT .... SUCCESS [  0.214 s]
[INFO] Helidon Examples Security Digest Authentication 4.1.0-SNAPSHOT SUCCESS [  0.314 s]
[INFO] Helidon Examples Security HTTP Signatures 4.1.0-SNAPSHOT SUCCESS [  0.415 s]
[INFO] Helidon Examples Translator Backend 4.1.0-SNAPSHOT . SUCCESS [  0.193 s]
[INFO] Helidon Examples Translator Frontend 4.1.0-SNAPSHOT  SUCCESS [  0.344 s]
[INFO] Helidon Examples WebClient Standalone 4.1.0-SNAPSHOT SUCCESS [  0.312 s]
[INFO] Helidon Examples WebServer Basic 4.1.0-SNAPSHOT .... SUCCESS [  0.193 s]
[INFO] Helidon Examples WebServer Basics 4.1.0-SNAPSHOT ... SUCCESS [  0.282 s]
[INFO] Helidon Examples WebServer CommentsAAS 4.1.0-SNAPSHOT SUCCESS [  0.249 s]
[INFO] Helidon Examples WebServer Echo 4.1.0-SNAPSHOT ..... SUCCESS [  0.146 s]
[INFO] Helidon Examples WebServer Imperative 4.1.0-SNAPSHOT SUCCESS [  0.133 s]
[INFO] Helidon Examples WebServer Multiple Ports 4.1.0-SNAPSHOT SUCCESS [  0.386 s]
[INFO] Helidon Examples WebServer Mutual TLS 4.1.0-SNAPSHOT SUCCESS [  0.283 s]
[INFO] Helidon Examples WebServer Observe 4.1.0-SNAPSHOT .. SUCCESS [  0.269 s]
[INFO] Helidon Examples WebServer TLS 4.1.0-SNAPSHOT ...... SUCCESS [  0.254 s]
[INFO] Helidon Examples WebServer WebSocket 4.1.0-SNAPSHOT  SUCCESS [  0.364 s]
```

Reply:
fixed